src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+*{
+  margin: 0;
+  }
```

Comment:
Fix code style everywhere
```suggestion
* {
  margin: 0;
}
```

Reply:
[ось](https://github.com/mate-academy/layout_stars/blob/master/checklist.md) чекліст по тому, як має виглядати код. Важливо притримуватись цих рекомендацій - адже це не тільки покращує читабельність, а й коли в командних проєктах налаштовані певні конфігурації - дасть змогу уникати постійних помилок